### DISCUSSION TITLE
Mapping Error?

### DISCUSSION THREAD

**Discussion by efaden** [meisnate12/Plex-Meta-Manager#250] [Q&A] [answered]
Trying to get this all setup and am seeing a lot of these types of errors:

| Mapping Error | plex://movie/<LONGNUMBER>          | No ID to convert for <NAME OF MOVIE> |

Suggestions how to fix this?

-Eric

**Comment by meisnate12** [comment, marked as answer]
You have to refresh the metadata for those movies 

When plex first came out with their new movie agent it didn't include IMDb or TMDb IDs in the metadata. You probably still have that metadata so it just needs to be refreshed

**Reply by efaden** [reply]
Alright.  I'll give that a try.  Thanks!

**Reply by efaden** [reply]
That fixed it.  Thanks!

**Reply by meisnate12** [reply]
no problem!
